qiskit_dynamics/backend/dynamics_backend.py

```diff
@@ -871,16 +929,19 @@ def default_experiment_result_function(
         raise QiskitError(f"meas_level=={backend.options.meas_level} not implemented.")
 
 
-def _validate_run_input(run_input, accept_list=True):
+def _validate_run_input(run_input, accepted_types, accept_list=True):
```

Comment:
@DanPuzzuoli  As I am removing the ability to input signals in the `DynamicsBackend.solve` function, should I remove this change as well?
It makes the `validate_run_input` function more versatile but now lacks a usecase as the list of allowed types is the same for all functions calling it.

Reply:
Yeah if it isn't being used you can remove this change.